BUILD.md

```diff
@@ -25,51 +30,69 @@ You can install the platform artifacts in your local maven cache by running the
 mvn clean install -DskipTests
 ````
 
-Note that unit and integration tests are skipped because they take a while and because it is needed some preparation as indicated in the following section.
+Optionally you might need smoke tests package for running in servlet-containers tests, then you need to run
+```
+mvn clean install -DskipTests -Pproduction -Pnpm-it
+```
 
 ## Running tests
 
-### Run Smoke Tests in Saucelabs
-1. you need valid Saucelabs credentials for running the tests
-2. Change to the IT module folder and run:
+There are three modules with Integration Tests, for running each one you need to enable the corresponding profile.
+
+In Addition, tests include Collaboration Engine, you need to provide a [valid license](https://vaadin.com/collaboration#free-users) to run it by setting a maven property e.g:
+
 ```
-cd vaadin-platform-test
-mvn verify -Dsauce.user=your_username -Dsauce.sauceAccessKey=your_key
+-Dce.license='{"content":{"key":"XXX","owner":"foo","quota":1000000,"endDate":"date"},"checksum":"XXX"}'
 ```
 
-### Run Smoke Tests in local computer
+### Run Integration Tests in Saucelabs
+1. you need valid Saucelabs credentials for running the tests
+2. For Smoke Integration Tests run:
+```
+mvn verify -Pproduction,npm-it -Dsauce.user=your_username -Dsauce.sauceAccessKey=your_key -Dce.license=your_ce_license
+```
+2. For Fusion Integration Tests run:
+```
+mvn verify -Pproduction,fusion-hybrid -Dsauce.user=your_username -Dsauce.sauceAccessKey=your_key -Dce.license=your_ce_license
+```
+3. For Servlet Contaner Integration Tests need to run:
+```
+mvn install -Pproduction,npm-it -DskipTests
+mvn verify -Pproduction,platform-servlet-containers-tests -Dsauce.user=your_username -Dsauce.sauceAccessKey=your_key -Dce.license=your_ce_license
+```
 
-1. Download the chrome driver appropriated for your system. Visit https://chromedriver.chromium.org/downloads
-2. Unzip the file and you will have the `chromedriver` in the current folder
-3. Change to the IT module folder and run:
+### Running Integration Tests in local computer
 
+1. For Flow Integration Tests run:
+```
+mvn verify -Pproduction,npm-it -Dce.license=your_ce_license
```

Comment:
better add `-Dcom.vaadin.testbench.Parameters.testsInParallel=1` to the command for better performance.